dagql/idtui/golden_test.go

```diff
@@ -35,33 +33,19 @@ import (
 	"github.com/dagger/dagger/dagql/idtui"
 	"github.com/dagger/dagger/engine/slog"
 	"github.com/dagger/dagger/internal/testutil"
-	"github.com/dagger/dagger/testctx"
+	"github.com/dagger/testctx"
+	"github.com/dagger/testctx/otelmw"
```

Comment:
out of context, "mw" doesn't immediately read as "middleware"... it makes sense to alias as that but idk about having that as the package name

Reply:
How about `oteltest`? Looks like it's largely unclaimed - just a some unused packages, and an official but deprecated package in a module that you'd have to go out of your way to install: https://pkg.go.dev/search?q=oteltest